packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilter.tsx

```diff
@@ -157,27 +157,38 @@ function EnumFacetFilterInternal(props: EnumFacetFilterProps) {
     )
     const valueNotSetFacetArray = partitions[0]
     const restOfFacetValuesArray = partitions[1]
-    let sortedValues: RenderedFacetValue[]
-    if (isNumberColumnType) {
-      sortedValues = sortBy(restOfFacetValuesArray, fv => Number(fv.value))
-    } else {
-      sortedValues = sortBy(restOfFacetValuesArray, fv =>
-        fv.displayText.toLowerCase(),
-      )
+
+    // Apply client-side sorting if no server-side sort is specified
+    let sortedValues: RenderedFacetValue[] = restOfFacetValuesArray
+    const isClientSideSort =
```

Comment:
In your opinion, would it be feasible to remove this code in the near future (after portal data managers have a chance to update their table schemas)?